integrations/pgvector/pydoc/config.yml

```diff
@@ -4,7 +4,6 @@ loaders:
     modules: [
       "haystack_integrations.components.retrievers.pgvector.embedding_retriever",
       "haystack_integrations.document_stores.pgvector.document_store",
-      "haystack_integrations.document_stores.pgvector.filters",
```

Comment:
this module contains the filtering logic for Pgvector.
Should not be user-facing.